src/styles/card_style.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
```

Comment:
instead of using 'margin-left' you could use display:  flex, align-self: center

Reply:
Ok, thanks for advice)